identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp

```diff
@@ -399,13 +399,18 @@
                                 %>
                                     <div class="two fields">
                                         <div id="firstNameField" class="field">
-                                            <label class="control-label"><%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "First.name")%>*</label>
+                                            <label class="control-label">
+                                                <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "First.name")%>
+                                                <% if (firstNamePII.getRequired() || !piisConfigured) { %>*<% } %>
+                                            </label>
                                             <input id="firstNameUserInput" type="text" name="http://wso2.org/claims/givenname" class="form-control"
                                                 <% if (firstNamePII.getRequired() || !piisConfigured) {%> required <%}%>
                                                 <% if (skipSignUpEnableCheck && StringUtils.isNotEmpty(firstNameValue)) { %> disabled <% } %>
                                                 <% if (StringUtils.isNotEmpty(firstNameValue)) { %>
                                                 value="<%= Encode.forHtmlAttribute(firstNameValue)%>"<% } %>
-                                                placeholder="<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "First.name")%>*"/>
+                                                placeholder="<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "First.name")%>
+                                                <% if (firstNamePII.getRequired() || !piisConfigured) { %>*<% } %>"
```

Comment:
We don't need * for placeholder, do we?

Reply:
I also assume it is not required. It was there previously